docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -1,68 +1,73 @@
+---
+description: How to spin up a Kubernetes cluster with Gardener
+---
```

Comment:
Sorry @colder-is-better, I meant to include this bit in the patch from the get-go but somehow forgot. I can haz re-approval plz?